js/data/create_bucket.js

```diff
@@ -15,6 +16,10 @@ function createBucket(layer, buffers, collision, indices) {
         return;
     }
 
+    for (var k in layer.layout) {
+        layer.layout[k] = new StyleDeclaration('layout', layer.type, k, layer.layout[k]).value;
```

Comment:
Oh, right, z14 is the max for most VT sources. So the bug is that it appears rotated at z18. It's only ever loading z14 tiles, so only the first stop value should be used.